__tests__/api.test.js

```diff
@@ -103,18 +94,47 @@ describe("/api/articles/:article_id", () => {
     });
 });
 
-/*
-
-an article object, which should have the following properties:
-author
-title
-article_id
-body
-topic
-created_at
-votes
-article_img_url
-
-
-
-*/
+describe("/api/articles", () => {
+    describe("GET", () => {
+        test("200: Responds with an array of article objects, each of which has appropriate properties (excluding comment_count which is next test)", () => {
```

Comment:
Nice test :)
Honestly, I would just check that comment count exists in this test and then check the value is correct on getArticleById but if you want to leave it like this then I think that'd be fine too. As long as you add to your description that the articles are sorted, then you could also check the sorting in this test too. It's your choice though.